x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/use_results_rollup/index.tsx

```diff
@@ -53,15 +60,58 @@ interface UseResultsRollup {
   updatePatternRollup: (patternRollup: PatternRollup) => void;
 }
 
+const useStoredPatternRollups = (patterns: string[]) => {
+  const { httpFetch, toasts } = useDataQualityContext();
+  const [storedRollups, setStoredRollups] = useState<Record<string, PatternRollup>>({});
+
+  useEffect(() => {
+    let ignore = false;
+    const abortController = new AbortController();
+
+    const fetchStoredRollups = async () => {
+      const results = await getResults({ httpFetch, abortController, patterns, toasts });
+      if (results?.length && !ignore) {
+        setStoredRollups(Object.fromEntries(results.map(({ rollup }) => [rollup.pattern, rollup])));
+      }
+    };
+
+    fetchStoredRollups();
+    return () => {
+      ignore = true;
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
> `React Hook useEffect has missing dependencies: 'httpFetch' and 'toasts'`

Consider returning stable references from the `useDataQualityContext` hook as an alternative to disabling `react-hooks/exhaustive-deps`